app/controllers/slack_controller.rb

```diff
@@ -25,6 +31,12 @@ def plus
         render json: {text: "Nope... not gonna happen."}
       end
     end
+  rescue MissingSlackToken
```

Comment:
exceptions for such situations is totally ok, but not when it's inside the same layer (and even more not, when it's in the same method.

I know that it was done this way before with `CannotPlusOneYourself`, but it's better if we move the whole logic from here into a service object and only then use exceptions to communicate the problem.